src/api/transaction/sign.js

```diff
@@ -48,12 +48,14 @@ function computeSignature(txJSON, keypair) {
 }
 
 function sign(txJSON: {Account: string; SigningPubKey: string,
-    TxnSignature: string}, secret: string):
+    TxnSignature: string}, secret: string, isRegular = false):
```

Comment:
We can. I just don't know what difference it make for transactions to be all signed with `get_key()` instead of `get_key(address)`, so I left it intact.
But it works, I've checked.